### PR TITLE
net-dns/dnscap: 1.10.3 bump and cleanup

### PR DESCRIPTION
**Pull request by dheule**

### CODE REVIEW

File: net-dns/dnscap/dnscap-1.10.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+DESCRIPTION="Network capture utility designed specifically for DNS traffic"
+HOMEPAGE="https://dnscap.dns-oarc.net/"
+SRC_URI="https://www.dns-oarc.net/files/dnscap/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="ISC"
+IUSE=""
+
+RDEPEND="dev-libs/openssl:=
```
Comment (juippis) [inline net-dns/dnscap/dnscap-1.10.3.ebuild]: Could you add a slot here, I think `:0=` is the correct syntax.

File: net-dns/dnscap/dnscap-1.10.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	dev-perl/YAML
+	net-libs/ldns
```
Comment (juippis) [inline net-dns/dnscap/dnscap-1.10.3.ebuild]: The latest version has a subslot, so if this library gets linked during compile, you should add `:=` here.

Comment (dheule) [reply net-dns/dnscap/dnscap-1.10.3.ebuild]: fixed

File: net-dns/dnscap/dnscap-1.10.3.ebuild (hunk 1 above)
Comment (dheule) [reply net-dns/dnscap/dnscap-1.10.3.ebuild]: changed to use the 0 slot of openssl

**Comment by juippis** [comment]
Thanks!

**Pull request closed without merging**